int8 id
int8 cluster
int16 x1
int16 x2
int16 y1
int16 y2
float64 xc
float64 yc
int8 life
bool vertical
